### PR TITLE
Facia tool breaking news

### PR DESCRIPTION
**Pull request by piuccio**
Visual changes for breaking news
- Background is now red
- Cannot change front
- Mobile indicators are hidden
- Link to breaking news in the landing page
- Rename some links (`draft front` -> `alerts`, `launch` -> `send alert`)

New features
- Show history of collections (last 5 items)
- Confirmation dialog before sending alerts

Breaking news looks like this
screen shot 2015-01-13 at 12 32 25

Alert
screen shot 2015-01-13 at 12 32 31

Other fronts
screen shot 2015-01-13 at 12 32 09

### CODE REVIEW

File: facia-tool/public/css/style.css [hunk 1]
```diff
@@ -1769,6 +1761,119 @@ collection-widget {
...
+    margin-right: 8px;
+}
+
+.modalDialog-confirm,
+.modalDialog-alert {
+    height: 42px;
+    padding: 0 6px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```
Comment (uplne) [inline facia-tool/public/css/style.css]: I like the use of flexbox :-)

**Comment by uplne** [comment]
:+1: ...minus points for no unit tests

File: facia-tool/app/views/collections.scala.html [hunk 2]
```diff
@@ -227,6 +236,35 @@
...
     </script>
 
+    <script type="text/html" id="confirm_breaking_changes">
+        <div class="modalDialog-message">
+            <h3>Confirm alert before sending</h3>
+            <ul data-bind="foreach: articles">
+                <li data-bind="html: headline"></li>
+            </ul>
+            <h4>Edition</h4>
+            <span class="edition" data-bind="text: edition"></span>
```
Comment (stephanfowler) [inline facia-tool/app/views/collections.scala.html]: Use 'target' instead of 'edition'

**Comment by stephanfowler** [comment]
:+1:

**Comment by phamann** [comment]
Merge conflict. Ship me!

**Comment by piuccio** [comment]
Merge conflict fixed, I also realized there was a problem when dragging snaps from the history, fixed as well.

**Comment by piuccio** [comment]
I promise this is the last commit on this PR :tongue: 

Character count backwards, @stephanfowler do you agree?

I'd love to ship this

**Comment by stephanfowler** [comment]
:+1: go 4 it

**Pull request merged**